Resources/doc/reference/configuration/user-context.rst

```diff
@@ -120,6 +120,16 @@ request. However, when you decide to cache hash responses, you must invalidate
 them when the user context changes, particularly when the user logs in or out.
 This bundle provides a logout handler that takes care of this for you.
 
+.. _always_vary_on_context_hash:
+
+``always_vary_on_context_hash``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**type**: ``boolean`` **default**: ``true``
+
+Set ``always_vary_on_context_hash`` to  ``false`` to disable always adding X-User-Context-Hash or Cookie, Authorization in Vary header.
```

Comment:
Actually that is not true anymore because, if you look at the event listener changes I made, when the option is set to false the Vary header is not modified in any case (i.e, whether the hash is present in the request or not). So the Vary Cookie, Auhtorization which is usually set by this bundle  will not be set anymore.